syntax = "proto3";

package sofa.pbrpc.test;

option cc_generic_services = true;
option java_generic_services = true;

message EchoRequest {
    string message = 1;
}

message EchoResponse {
    string message = 1;
}

service EchoServer {
    rpc Echo(EchoRequest) returns(EchoResponse);
}
